tests/integration-service/integration.go

```diff
@@ -107,6 +109,17 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 				return nil
 			}, timeout, constants.PipelineRunPollingInterval).Should(Succeed(), fmt.Sprintf("timed out when waiting for the PipelineRun to start for the component %s/%s", testNamespace, componentName))
 			Expect(f.AsKubeDeveloper.HasController.WaitForComponentPipelineToBeFinished(originalComponent, "", 2)).To(Succeed())
+			Expect(pipelineRun.Annotations["appstudio.openshift.io/snapshot"]).To(Equal(""))
+		}
+
+		assertBuildPipelineRunSigned := func() {
+			Eventually(func() error {
+				pipelineRun, _ := f.AsKubeAdmin.IntegrationController.GetBuildPipelineRun(componentName, applicationName, testNamespace, false, "")
```

Comment:
ignoring the potential error here will lead to a runtime error on the line below

Reply:
OK.  It's added back.